pkg/operator/resources/errors.go

```diff
@@ -61,3 +64,17 @@ func ErrorNoAvailableNodeComputeLimit(resource string, reqStr string, maxStr str
 		Message: message,
 	})
 }
+
+func ErrorAPIUsedByAPISplitter(apiSplitters []string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrAPIUsedByAPISplitter,
+		Message: fmt.Sprintf("can not delete api because it is used by APISplitter: %s", strings.StrsSentence(apiSplitters, "")),
```

Comment:
`can not delete api because it is used by APISplitter:` -> `cannot delete api because it is used by the following APISplitter:` and use pkg/lib/strings.PluralS() to make `APISplitter` plural if apiSplitters is more than 1 and